frog/imports/client/App/TopBar.js

```diff
@@ -115,7 +115,7 @@ class TopBarController extends React.Component<
 > {
   routes = [
     { name: 'Graph Editor', to: '/teacher/graph' },
-    { name: 'Sessions', to: '/teacher/orchestration' },
+    { name: 'Sessions', to: '/t/' },
```

Comment:
Careful, this path is the only path with a `/` at the end of the string. Make sure this is not a typo

Reply:
Both are equally valid, will change for consistency